packages/engine-server/src/workspace/service.ts

```diff
@@ -740,6 +742,56 @@ export class WorkspaceService {
     );
   }
 
+  /**
+   * See if there's anythign we need to change with the configuration
+   */
+  async runMigrationsIfNecessary({
+    forceUpgrade,
+    workspaceInstallStatus,
+    currentVersion,
+    previousVersion,
+    dendronConfig,
+  }: {
+    forceUpgrade?: boolean;
+    workspaceInstallStatus: InstallStatus;
+    currentVersion: string;
+    previousVersion: string;
+    dendronConfig: DendronConfig;
+  }) {
+    // check if we need to force a migration
+    try {
+      const maybeRaw = DConfig.getRaw(this.wsRoot);
+      if (
+        _.isUndefined(maybeRaw.journal) &&
+        workspaceInstallStatus === InstallStatus.INITIAL_INSTALL
```

Comment:
wouldn't all the workspaces with `InstallStatus.INITIAL_INSTALL` have the `journal` config in there by default?

Reply:
you're right. updated